packages/react-aria-components/docs/Table.mdx

```diff
@@ -1081,6 +1091,7 @@ function Example() {
 .react-aria-Row {
   &[data-dragging] {
     opacity: 0.6;
+    transform: translateZ(0);
```

Comment:
Why this one?

Reply:
same reason as this one https://github.com/adobe/react-spectrum/pull/4333/files#r1157714584